src/app/modules/feedback/__tests__/feedback.service.spec.ts

```diff
@@ -248,4 +248,64 @@ describe('feedback.service', () => {
       expect(actualResult._unsafeUnwrapErr()).toBeInstanceOf(DatabaseError)
     })
   })
+
+  describe('checkHasPreviousFeedback', () => {
+    const MOCK_FORM_ID = new ObjectId().toHexString()
+    const MOCK_SUBMISSION_ID = new ObjectId().toHexString()
+
+    beforeEach(async () => {
+      await dbHandler.clearCollection(FormFeedback.collection.name)
+    })
+    afterEach(() => jest.clearAllMocks())
+
+    it('should return true when previous feedback exists', async () => {
```

Comment:
the test description isn't very clear - maybe something like `should return true when feedback already exists for given submision ID`?